plugin-core/src/main/java/appland/files/AppMapAsyncFileListener.java

```diff
@@ -40,17 +40,6 @@ public class AppMapAsyncFileListener implements AsyncFileListener {
     }
 
     private static boolean isAppMapFileChange(@NotNull VFileEvent event) {
-        if (event.isFromRefresh()) {
```

Comment:
The intention was to let `appland.files.VirtualFileManagerLister` notify after a filesystem refresh finished and this listener to handle non-refresh events. But this split approach is not suitable if you need to know about e.g. new files picked up by a refresh.

I'd remove the note of this class's documentation about the now removed refresh handling.

Reply:
This change could lead to multiple notifications about refresh events, e.g. once for a new file from a refresh and again by `VirtualFileManagerLister` after the refresh finished.
I checked the consumers of the notifications, they're debouncing so this change shouldn't make it flicker, for example.

Reply:
In practice I was only observing new files picked up by refresh. I'm not clear on the conditions under which a file event which did not originate from a refresh would occur.